mlir/test/Dialect/Vector/canonicalize.mlir

```diff
@@ -3149,6 +3149,18 @@ func.func @contiguous_gather_step(%base: memref<?xf32>,
 
 // -----
 
+// CHECK-LABEL: @dont_fold_tensor_type_contiguous_gather
+func.func @dont_fold_tensor_type_contiguous_gather(%base: tensor<8xf32>, %mask: vector<4xi1>, %pass_thru: vector<4xf32>) -> vector<4xf32> {
```

Comment:
1. `no_fold` is shorter than `dont_fold` :)
2. The naming format seems to be `@contiguous_gather_{other_stuff}`, so `@dont_fold_tensor_type_contiguous_gather` -> `@no_fold_contiguous_gather_tensor` (or something similar)

Reply:
Changed it to `@no_fold_contiguous_gather_tensor`.